sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -143,6 +148,24 @@ above.
 
 # Use SYCL toolchain
 
+## Using the SYCL toolchain on CUDA platforms
+
+The SYCL toolchain support on CUDA platforms is still on an experimental phase.
+Currently, the SYCL toolchain relies on having a recent OpenCL implementation
+on the system in order to link applications to the SYCL runtime.
+The OpenCL implementation is not used at runtime if only the CUDA backend is
+active for the application.
```

Comment:
The OpenCL implementation is not used at runtime if only the CUDA backend is active for the application but must be installed.

Reply:
Addressed in 7d027efeda7845a8d6df6f7fa109d4c73451ea61